code/clue.js

```diff
@@ -1,59 +1,317 @@
+  name: 'Garden',
+  color: 'PapayaWhip',
+  image: 'assets/room/garden.png'
+};
+
+
+
+// ARRAYS FOR SUSPECTS, WEAPONS AND ROOMS //
+
+//Array for suspects
 const suspects = [
   mrGreen,
+  colMustard,
+  mrsPeacock,
+  profPlum,
+  missScarlet,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+//Array for weapons
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
 
-const rooms = []
+// Array for rooms
+const rooms = [
+  diningRoom,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  spa,
+  livingRoom,
+  theater,
+  garden
+];
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
+//FUNCTION FOR RANDOMLY SELECTING ONE ITEM FROM THE ARRAY THAT'S PASSED INTO THE FUNCTIONS FOR SUSPECTS, WEAPONS AND ROOMS.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+//OBJECT THAT KEEPS THE MYSTERY //
+const mystery = {
+ killer: null,
+ weapon: null,
+ room: null
+};
+
+
+// FUNCTIONS FOR CARDS //
+
+//Function to show hidden cards
+const cardShow = () => {
+  const element = document.getElementById("cardShow");
+  element.classList.remove('cardShow');
+} 
+document.getElementById('startPlay').onclick = cardShow;
+
 
 // This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
-
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  mystery.killer = randomSelector(suspects)   // This will randomly select a killer from the suspects. And add that to the mystery object.
+  document.getElementById('killerCard').style.background = mystery.killer.color   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer.
+  document.getElementById('killerName').innerHTML = `Name: ${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerAge').innerHTML = `Age: ${mystery.killer.age}`
+  document.getElementById('killerOccupation').innerHTML = `Occupation: ${mystery.killer.occupation}`
+  document.getElementById('killerImage'); killerImage.src = mystery.killer.image
+  document.getElementById('killerDescription').innerHTML = `${mystery.killer.description}`;
```

Comment:
In todays class Maks told us it was considered more clean code to create const variables for this kind of elements.
I.e. 
`const killerAGe = document.getElementById('killerAge').innerHTML;
killerAge = `Age: ${mystery.killer.age}`;`

Reply:
Let me clarify on that :
If you will use them more than one time - yes.
If only one time - no.